src/libs/PolicyUtils.ts

```diff
@@ -897,6 +897,15 @@ function getCurrentTaxID(policy: OnyxEntry<Policy>, taxID: string): string | und
     return Object.keys(policy?.taxRates?.taxes ?? {}).find((taxIDKey) => policy?.taxRates?.taxes?.[taxIDKey].previousTaxCode === taxID || taxIDKey === taxID);
 }
 
+function getWorkspaceAccountID(policyID: string) {
+    const policy = getPolicy(policyID);
+
+    if (!policy) {
+        return '';
+    }
+    return policy.workspaceAccountID ?? 0;
```

Comment:
@koko57 I am curious about the difference between `policyID` and `workspaceAccountID`. Could you help to explain this? 

cc @mountiny

Reply:
we use workspaceAccountID instead of policyID for the expensifyCard identifier

Reply:
I'm asking this because I noticed that in the functions updateExpensifyCardLimit and updateSettlementFrequency, we define workspaceAccountID, but when calling these functions, we pass policyID.

Reply:
@DylanDylann They should pass workspaceAccountID, I should have corrected them too - I can do it in this PR or I will do it while integrating with the BE. workspaceAccountID and policyID have different values

Reply:
They are different IDs and are used for different concepts in the backend.

It's a bit complicated, and it's built like that for historical reasons, but the card functionality is tied to "domain." A domain is a row in the accounts table (the same as using an account when they sign up). The domain accounts are specific given the email format they use. 

Now to have an expensify card in oldDot you need to have a verified domain (like expensify.com). We are changing that for the workspace feeds, but we still create this fake account in the database that is 1:1 tied to the policy. The workpaceAccountID is the accountID of this "fake" domain account. policyID is just an ID of the policy